classes/blocks/class-carouselheader.php

```diff
@@ -22,17 +22,34 @@ class CarouselHeader extends Base_Block {
 	 *
 	 * @const string BLOCK_NAME.
 	 */
-	const BLOCK_NAME = 'carousel_header';
+	const BLOCK_NAME = 'carousel-header';
+
+	/**
+	 * Block category.
+	 *
+	 * @const string BLOCK_CATEGORY.
+	 */
+	const BLOCK_CATEGORY = 'planet4-blocks';
```

Comment:
I don't think there's a reason to split this across 2 constants, as the block name is always Namespace + Name, we never need to use these separately. Also "category" is confusing, since we also have a category on the front end.
https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/b2ad5d4448aa089a887faa2b7d85af95e781812e/assets/src/blocks/CarouselHeader/CarouselHeaderBlock.js#L46

Reply:
This category happens to be the same string as the Namespace, but it could be another one like here https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/b2ad5d4448aa089a887faa2b7d85af95e781812e/assets/src/blocks/Accordion/AccordionBlock.js#L40-L40

Reply:
Ah, I just needed to split the name so I could automatically map it into something like "CarouselHeader" in the filesystem. I could split it when I need to, but I also thought separating them would be more accurate to not add a prefix to the name, as the "name" of the block is `carousel-header` and not `planet4-blocks/carousel-header`.

Reply:
I previously glossed over the fact that the name wasn't prefixed before, thought you had split it, my bad :facepalm:

Reply:
Isn't this currently breaking some twig templates? https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/6e7af53bffbd965f1ecf32906a606900224f752f/classes/blocks/class-base-block.php#L42

Reply:
This was updated in the [split blocks PR](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/508), in a "let's call a spade a spade" basis: `BLOCK_NAME` is just the name, `BLOCK_NAMESPACE_PREFIX` is the namespace. I needed to separate them to be able to use the name for a few defaults without needing to split it.